src/mina-signer/copied/curve.ts

```diff
@@ -0,0 +1,14 @@
+import { Bool, Field } from './field.js';
```

Comment:
note: don't review anything in `src/mina-signer` for this PR. this stuff will be integrated in a proper form in snarkyjs piece by piece, but the versions here are in large parts redundant with snarkyjs code or copied from the old mina-signer